src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectSettingsController.java

```diff
@@ -44,44 +41,142 @@
 		this.messageSource = messageSource;
 		this.metadataTemplateService = metadataTemplateService;
 		this.projectControllerUtils = projectControllerUtils;
 		this.projectService = projectService;
 		this.projectRemoteService = projectRemoteService;
 		this.userService = userService;
+		this.analysisSubmissionService = analysisSubmissionService;
+		this.workflowsService = workflowsService;
 	}
 
 	/**
 	 * Request for a {@link Project} basic settings page
 	 *
-	 * @param projectId
-	 *            the ID of the {@link Project} to read
-	 * @param model
-	 *            Model for the view
-	 * @param principal
-	 *            Logged in user
-	 *
+	 * @param projectId the ID of the {@link Project} to read
+	 * @param model     Model for the view
+	 * @param principal Logged in user
+	 * @param locale    Locale of the logged in user
 	 * @return name of the project settings page
 	 */
 	@RequestMapping("")
 	public String getProjectSettingsBasicPage(@PathVariable Long projectId, final Model model,
-			final Principal principal) {
+			final Principal principal, Locale locale) {
 		Project project = projectService.read(projectId);
+		List<AnalysisSubmissionTemplate> templates = analysisSubmissionService.getAnalysisTemplatesForProject(project);
+
+		List<TemplateResponseType> templateResponseTypes = templates.stream()
+				.map(t -> templatesToResponse(t, locale))
+				.collect(Collectors.toList());
+
 		model.addAttribute("project", project);
 		model.addAttribute(ProjectsController.ACTIVE_NAV, ACTIVE_NAV_SETTINGS);
 		model.addAttribute("page", "basic");
+		model.addAttribute("analysisTemplates", templateResponseTypes);
 		projectControllerUtils.getProjectTemplateDetails(model, principal, project);
 		return "projects/settings/pages/basic";
 	}
 
+	/**
+	 * Convert a analysis template to {@link TemplateResponseType}
+	 *
+	 * @param template the {@link AnalysisSubmissionTemplate}
+	 * @param locale    User's logged in locale
+	 * @return a list of {@link TemplateResponseType}
+	 */
+	private TemplateResponseType templatesToResponse(AnalysisSubmissionTemplate template, Locale locale) {
+			UUID workflowId = template.getWorkflowId();
+			String typeString;
+
+			try {
+				IridaWorkflow iridaWorkflow = workflowsService.getIridaWorkflow(workflowId);
+				AnalysisType analysisType = iridaWorkflow.getWorkflowDescription()
+						.getAnalysisType();
+
+				typeString = messageSource.getMessage("workflow." + analysisType.getType() + ".title", null, locale);
+			} catch (IridaWorkflowNotFoundException e) {
+				typeString = messageSource.getMessage("workflow.UNKNOWN.title", null, locale);
+			}
+
+			return new TemplateResponseType(template.getId(), template.getName(), typeString);
+
+	}
+
+	/**
+	 * Response class for easily formatting analysis templates for the project settings page
+	 */
+	private class TemplateResponseType {
```

Comment:
Can you move this into its own file, I think this controller is big enough without this in it.  Peter started creating packages called `dto` (data transfer objects - I think) that he started putting them.  I have tried to keep the same naming (I need to go back and clean up the ones that are in packages called `models` as I think this gets confusing with our other models.

Reply:
Done in e8c5deb